File: source/spring-petclinic-rest/src/main/java/com/spring_petclinic/spring_petclinic_rest/application/models/OwnerUpdateDTO.java
package com.spring_petclinic.spring_petclinic_rest.application.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class OwnerUpdateDTO {
    private String firstName;
    private String lastName;
    private String address;
    private String city;
    private String telephone;

}

File: source/nestjs-petclinic-rest/src/services/dto/owner-update.dto.ts
import { ApiProperty } from '@nestjs/swagger';

export class OwnerUpdateDTO {
  @ApiProperty()
  firstName: string;

  @ApiProperty()
  lastName: string;

  @ApiProperty()
  address: string;

  @ApiProperty()
  city: string;

  @ApiProperty()
  telephone: string;
}